Base.rte/Scripts/GlobalScripts.ini

```diff
@@ -32,7 +32,7 @@ AddGlobalScript = GlobalScript
 
 AddGlobalScript = GlobalScript
 	PresetName = Tough Craft
-	Description = Crafts are nearly impossible to take down
+	Description = Craft are nearly impossible to take down
```

Comment:
missed a period, i'd edit it for you if i knew how

Reply:
Right, the global script grammar is WIP because I started doing them while moving my changes from B33 over to CCCP, I'll focus on those later.